lib/combineNodes.js

```diff
@@ -0,0 +1,367 @@
+'use strict';
+var Cesium = require('cesium');
+var jp = require('jsonpath');
+var _ = require('underscore');
```

Comment:
Yup, you're right. I thought maybe they were doing a randomized search, which is more optimal than linear for non-sorted data, but it is just linear.